src/lib/layers/layers-counter.component.ts

```diff
@@ -8,23 +8,34 @@ import { FaLayersComponent } from './layers.component';
   selector: 'fa-layers-counter',
   template: '',
   host: {
-    class: 'ng-fa-layers-counter',
+    class: 'fa-layers-counter',
```

Comment:
Why is this change needed?

Reply:
Sorry, my bad. I think it could back as normal without problem!